Documentation/Signoffs/Ground Based Sensor Subsystem.md

```diff
@@ -0,0 +1,215 @@
+# Ground-Based Sensor Subsystem
+
+The ground based sensor subsystem is to sense vehicles passing above the sensor, allowing for the system to keep a count of the
+number of cars that are in a specified parking lot on the campus of Tennessee Tech.  This system will be placed at parking lots
+that have specified entrances/exits.  By using lots that have this constraint, it will allow the system to keep an accurate count
+of the number of cars in a specific lot.
+
+# Constraints
+| No. | Constraints | Origin |
+| --- | ----------- | ------ |
+| 1.  | Subsystem shall detect vehicles entering and exiting a parking lot. | System Requirements |
```

Comment:
Not sufficiently precise. vehicle? entering? exiting?

Reply:
I have updated the Constraints section by adding numeric values to Constraint 1.  I have also defined what the term vehicle means in the updated markdown file.